### DISCUSSION TITLE
No DXCC on "Create Station Location" in a new fresh Installation

### DISCUSSION THREAD

**Discussion by ea5wa** [magicbug/Cloudlog#1078] [Q&A] [answered]
I'm installing Cloudlog on a new server. Everything is already installed, but when I try to create a new Station Location, I'm having an error message because there is no Station DXCC in the "Create Station Location" form...

![image](https://user-images.githubusercontent.com/54686689/123991891-65890880-d9cb-11eb-9f42-66151fe69913.png)

**Comment by ea5wa** [comment]
Trying to do it for 3 times, now is my Station Location created.... Without any textbox for Station DXCC in the form....

**Comment by ea5wa** [comment]
Trying to set a second location, same problem....

![image](https://user-images.githubusercontent.com/54686689/123993641-e563a280-d9cc-11eb-8185-4621f8e8c757.png)

**Comment by ea5wa** [comment, marked as answer]
It was my fault.... I forgot to Update Country Files

**Comment by 4O4A** [comment]
Is there any other way to solve this issue?

Clouldlog is trying to download file from ClubLog, but actually nothing is happening for last 6 hours or so.
![image](https://user-images.githubusercontent.com/101504692/210184067-a48f9005-1159-4336-84bd-5b2803acf00e.png)

**Reply by 4O4A** [reply]
... resulting with "... is currently unable to handle this request. HTTP ERROR 500" after some time.

**Reply by 4O4A** [reply]
Looks like temp problem in communication between my cloudlog and Clublog.
Exporting and importing four tables from another instance of Cloudlog, with "dxcc" prefix, temporary solved the problem.
